Formula/protobuf.rb

```diff
@@ -34,24 +25,34 @@ class Protobuf < Formula
   depends_on "cmake" => :build
   depends_on "python@3.10" => [:build, :test]
   depends_on "python@3.11" => [:build, :test]
+  depends_on "abseil"
 
   uses_from_macos "zlib"
 
+  # Fix unexported symbols on arm64.
+  patch do
+    url "https://github.com/Homebrew/formula-patches/commit/9f5bd2532efc119c0836f70bc9e8c5b3f5367715.patch"
+    sha256 "633472c9d1ec67597aa0d5862cab04710e0878c68dc3464b9e84fd48cd2f5ac4"
+  end
```

Comment:
```suggestion
  on_arm do
    patch do
      url "https://github.com/Homebrew/formula-patches/commit/9f5bd2532efc119c0836f70bc9e8c5b3f5367715.patch"
      sha256 "633472c9d1ec67597aa0d5862cab04710e0878c68dc3464b9e84fd48cd2f5ac4"
    end
  end
```
Let's keep this patch only on ARM.